README.md

```diff
@@ -427,9 +427,15 @@ newly-launched instance.
 > favourite training environment in the cloud by pulling the Docker container of
 > your choice.
 
-We like the CML container (`docker://dvcorg/cml-py3`) because it comes loaded
-with Python, CUDA, `git`, `node` and other essentials for full-stack data
-science.
+We like the CML container (`docker://dvcorg/cml`) because it comes loaded with
+Python, CUDA, `git`, `node` and other essentials for full-stack data science.
+Different versions of these essentials are available from different `dvcorg/cml`
+image tags. The tag convention is `{CML_VER}-dvc{DVC_VER}-base{BASE_VER}{-gpu}`:
+
+| `{BASE_VER}` | Software included (`-gpu`)                      |
+| ------------ | ----------------------------------------------- |
+| 0            | Ubuntu 18.04, Python 2.7 (CUDA 10.1, CuDNN 7)   |
+| 1            | Ubuntu 20.04, Python 3.8 (CUDA 11.0.3, CuDNN 8) |
```

Comment:
idk if this is clear enough or can be improved

Reply:
Nice! I've been playing with some alternatives, but none of them is as explicit as your proposed version.

```suggestion
We like the CML container (`docker://dvcorg/cml`) because it comes loaded with
Python, CUDA, `git`, `node` and other essentials for full-stack data science.
Different versions of these essentials are available from different `dvcorg/cml`
image tags with the `{CML}-dvc{DVC}-base{BASE}` naming convention, where
`{CML}` and `{DVC}` are numbers representing their respective major versions,
and `{BASE}` determines the version of the following system packages:

| Base  | Ubuntu | Python | CUDA*  | cuDNN* |
| ----- | ------ | ------ | ------ | ------ |
| **0** | 18.04  | 2.7    | 10.1   | 7      |
| **1** | 20.04  | 3.8    | 11.0.3 | 8      |

_* Only included with tags containing the optional `-gpu` suffix._
```

Reply:
👍  for the usage examples